charts/qdrant/templates/statefulset.yaml

```diff
@@ -50,6 +50,10 @@ spec:
         volumeMounts:
           - name: qdrant-storage
             mountPath: /qdrant/storage
+        {{- with .Values.containerSecurityContext }}
```

Comment:
Hi @Johannes- , thanks for the contribution!

As mentioned here: https://github.com/qdrant/qdrant-helm/issues/130 I think this would break by default since the init container needs to run as root, but using the default containerSecurityContext will make it run as user 1000.

Instead of re-using `containerSecurityContext`, could it be controlled by a new, separate setting called `initContainerSecurityContext`? May lead to duplication in peoples' values, but it allows people to assign different security contexts to each.

Reply:
Also in #177 I removed the second init container, consolidating them into a single init container. That's probably what the merge conflict is.